examples/Huggingface_Transformers/Transformer_handler_generalized.py

```diff
@@ -14,8 +15,7 @@
 from captum.attr import LayerIntegratedGradients
 
 logger = logging.getLogger(__name__)
-
-
+print('Transformers version',transformers.__version__)
```

Comment:
Please use the logger for logging vs using print

Reply:
Resolved.